python/ray/serve/tests/test_router.py

```diff
@@ -161,6 +161,7 @@ async def test_shard_key(serve_instance, task_runner_mock_actor):
             assert call.request_args[0] in runner_shard_keys[i]
 
 
+# TODO(fix)
```

Comment:
@edoakes @simon-mo this tests uses `ray.register_custom_serializer`, what's the best way to fix it?

Reply:
I'll remove it ASAP. on it

Reply:
It's gone